api/Makefile

```diff
@@ -53,6 +53,17 @@ gen: clean            ## Generate files.
 
 	swagger-order --output=swagger/swagger.json swagger/swagger.json
 
+	# generate dev API spec
+	swagger mixin --output=swagger/swagger-dev.json \
+		swagger/header-dev.json \
```

Comment:
can we try to mixin `swagger.json` with `dbaas.json` and check if the output is the same?
Also let's include `ia` and `qan` packages.

Reply:
Mixin swagger.json will not work as we need a custom header.
Added is and qan.